### PR TITLE
791 load specific stories

### PR DESCRIPTION
**Pull request by DaJansenGit**
This brings the implementation to load only specific stories, the next step will be to overwrite boundary conditions to top and bottom.

I did some tests on existing IFC Files:

#### Lower Storey
IFC:
image
Result IDF:
image

#### Upper Storey
IFC:
image
Result IDF:
image

### DigitalHub
* DigitalHub is problematic, as it has multiple multi storey components, but these are only assigned to one storey in the hierarchy.
* This leads to missing spaces and components in the resulting model
#### Roof Storey
In this example, only the roof storey was used:
IFC: 
image
Result IDF:
image

#### Middle Storey
IFC:
image
Result IDF:
image

### CODE REVIEW

File: bim2sim/sim_settings.py [hunk 1]
```diff
@@ -548,6 +617,14 @@ def check_mandatory(self):
                     'additional 2b space boundaries.',
         for_frontend=True
     )
+    stories_to_load_guids = GuidListSetting(
+        default=[],  # Initialize with empty list by default
+        description='List of IFC GUIDs for the specific stories that should be'
+                    ' loaded. If empty, all stories will be considered '
```
Comment (veronikarichter) [inline bim2sim/sim_settings.py:623]: Maybe add a hint here, that loading single storeys may lead to missing ceilings (if the related slab is assigned to the storey above) and that boundary conditions may result some corrections.
Also add that this setting may be used for large buildings to reduce computational time. It is recommended to enter guids of neighboring storeys to reduce the amount of boundary condition errors.

**Review by veronikarichter — CHANGES_REQUESTED** [review]

File: bim2sim/sim_settings.py (hunk 1 above)
Comment (DaJansenGit) [reply bim2sim/sim_settings.py]: You are right. I added some hints to the description

**Review by veronikarichter — APPROVED** [review]
